File: src/main/java/net/sharplab/epubtranslator/core/model/EPubContentFile.java
package net.sharplab.epubtranslator.core.model;

/**
 * EPubContentFile
 */
public class EPubContentFile {

    private final String name;
    private final byte[] data;

    public EPubContentFile(String name, byte[] data){
        this.name = name;
        this.data = data;
    }

    public String getName() {
        return name;
    }

    public byte[] getData() {
        return data;
    }

}

File: src/main/java/net/sharplab/epubtranslator/core/model/EPubContentFile.kt
package net.sharplab.epubtranslator.core.model

import net.sharplab.epubtranslator.core.util.ArrayUtil

/**
 * EPubContentFile
 */
open class EPubContentFile(val name: String, data: ByteArray) {

    val data: ByteArray = ArrayUtil.clone(data)
        get() = ArrayUtil.clone(field)

}